hack/docs/instancetypes_gen_docs.go

```diff
@@ -195,3 +199,47 @@ below are the resources available with some assumptions and after the instance o
 		}
 	}
 }
+
+type kubeDnsTransport struct {
+}
+
+const kubeDNS = `{
```

Comment:
Thoughts on ripping out most of this config and just configuring what's necessary for the mock to work?

Reply:
I spent a few minutes and minimized it to this from the full output.  When it broke, I added back what I had removed and quit trying.